lib/rubocop/cop/lint/number_conversion.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Lint
+      # This cop check usage of unsafe number conversions, eg. `foo.to_i`.
+      #
+      # @example
+      #
+      #   # bad
+      #   `foo.to_i`
```

Comment:
I'd refer to `to_f` and `to_c` in the examples as well.

Reply:
Added examples for `to_f` and `to_c`.